http-server/src/main/java/org/triplea/modules/user/account/login/AccessLogUpdater.java

```diff
@@ -21,15 +21,10 @@ public static AccessLogUpdater build(final Jdbi jdbi) {
   @Override
   public void accept(final LoginRecord loginRecord) {
     final int updateCount =
-        loginRecord.isRegistered()
-            ? accessLogDao.insertRegisteredUserRecord(
-                loginRecord.getUserName().getValue(),
-                loginRecord.getIp(),
-                loginRecord.getSystemId().getValue())
-            : accessLogDao.insertAnonymousUserRecord(
-                loginRecord.getUserName().getValue(),
-                loginRecord.getIp(),
-                loginRecord.getSystemId().getValue());
+        accessLogDao.insertUserAccessRecord(
```

Comment:
`insertUserAccessRecord` can now figure out on its own if a user is registered or not by selecting from the lobby_user table.